lightkurve/prf/tpfmodel.py

```diff
@@ -406,7 +406,7 @@ def __init__(self, star_priors=[],
                  focus_prior=FocusPrior(),
                  motion_prior=MotionPrior(),
                  prfmodel=KeplerPRF(1, shape=(10, 10), column=0, row=0),
-                 fit_background=True, fit_focus=False, fit_motion=False):
+                 fit_background=False, fit_focus=False, fit_motion=False):
```

Comment:
why not fit the background by default? I think it's a common thing to add the background back (to avoid negative flux values) and fit it (Libralato et. al. does that).